install.sh

```diff
@@ -423,12 +423,6 @@ fi
 # Default cmake executable is called cmake
 cmake_executable=cmake
 
-case "${ID}" in
-  centos|rhel)
-  cmake_executable=cmake3
```

Comment:
This is required on Centos as far as I know as most cmake installs are cmake3 not cmake.

Reply:
This was removed in rocBLAS-internal PR 616, so rocBLAS no longer has this change to cmake3 for centos and rhel. I will test it on centos and rhel.

Reply:
@daineAMD I guess you did that, do you recall that cmake3 was retired?   Or I guess if we are not going with the default anymore the new install is just cmake and not the default cmake3 (symlink) ?

Reply:
I unfortunately don't recall the details of removing cmake3, and don't know any details of cmake versioning. I think Alex ran into some issue on Centos with the oly cmake_executable=cmake3 in rocBLAS.

Reply:
there are come comments in https://ontrack-internal.amd.com/browse/SWDEV-311824   from Zhongyu saying cmake3 is an alias to cmake in Centos8

Reply:
Anyway, sounds fine to remove if done on rocblas they both either work or don't :-)   If you pull dockers you can see if Centos6 and 7 had cmake3 symlink to version 3.X and cmake was symlink to  2.X  ?   If user runs into trouble they can add a cmake3 symlink.